src/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+const useLocalStorage = (key: string, initialValue = '') => {
+  const [storageValue, setStorageValue] = useState(() => {
+    try {
+      const item = window.localStorage.getItem(key);
+      return item ? JSON.parse(item) : initialValue;
+    } catch (error) {
+      return initialValue;
+    }
+  });
+
+  const setValue = (value: unknown) => {
+    try {
+      window.localStorage.setItem(key, JSON.stringify(value));
+      setStorageValue(value);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
+  };
+
+  return [storageValue, setValue];
```

Comment:
사용하는 곳에서 type infer가 잘 되던가요? 👀 
`as const` 요 처리가 필요해보입니다만 ㅎㅎ

```suggestion
  return [storageValue, setValue] as const;
```

Reply:
으앗, 저건 const assertion으로 해결할 수 있었던 문제였군요! 궁금증이 해결됐습니다!
사실 저 코드는 더 이상 사용하지 않는 코드인데, 같이 올려버린 제 실수입니다 😅
피드백 감사드립니다!

Reply:
https://github.com/woowacourse/react-subway-map/pull/12/commits/ca30564293a03a44074691b29c615fac4d193517 에서 반영하였습니다!